src/options.rs

```diff
@@ -108,3 +101,21 @@ impl TryFrom<&Args> for Options {
         })
     }
 }
+
+fn build_glob_set(glob_set: &Vec<String>) -> Result<Option<GlobSet>>{
```

Comment:
even though it is fairly self-explanatory maybe add a comment?

Reply:
Maybe instead of adding a comment, we could also reveal more intent in the function name. 

How about this name: `build_glob_set_from_cli_arg_values` ?

Reply:
maybe it's fine as it is